.github/workflows/ci.yml

```diff
@@ -44,6 +44,18 @@ jobs:
         ini-values: apc.enable_cli = 1
         coverage: pcov
 
+    - name: Get composer cache directory
+      id: composer-cache
+      run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+    - name: Cache composer dependencies
+      uses: actions/cache@v1
+      env:
+        KEY_DATE=$(date '+%y-%m')
```

Comment:
Why is this one different than the others?

Reply:
It's still WIP. I still haven't figured how to get the required date string is a variable which I can use in the next step :)